cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -270,25 +326,28 @@ my-log-viewer {
           color: #7f7f7f;
           position: relative;
           left: 7px;
-          top: 6px;
+          top: 10px;
         }
 
         .message-column {
           word-wrap: break-word;
           padding-left: 5px;
+          position: absolute;
+          top: 0;
+          left: 20px;
         }
 
         .stack-trace {
           position: relative;
-          padding-bottom: 20px;
-          bottom: 10px;
-          height: 100%;
+          padding-bottom: 11px;
+          height: 101%;
```

Comment:
I don't think we should set height to 101%. If this height needs to be >100%, then there's something wrong somewhere else